airflow-ctl/src/airflowctl/ctl/cli_config.py

```diff
@@ -192,6 +193,33 @@ def __call__(self, parser, namespace, values, option_string=None):
     nargs="?",
 )
 
+ARG_OUTPUT = Arg(
+    flags=("-o", "--output"),
+    type=str,
+    default="table",
+    choices=("table", "json", "yaml", "plain"),
```

Comment:
Let's delete the choices we don't support at the moment

Reply:
missed this, thanks 👍🏻